packages/portal/frontend/webpack.config.js

```diff
@@ -1,10 +1,50 @@
 var path = require('path');
 
+// read the env so we can copy custom resources (if needed)
+require('dotenv').config(
+    {path: '../../../.env'}
+);
+
+// copy plugin files so they are available to frontend
+const CopyPlugin = require('copy-webpack-plugin');
+
+// handle @frontend and @common type import aliases from plugin
+const TsconfigPathsPlugin = require('tsconfig-paths-webpack-plugin');
+
+console.log('Preparing frontend for: ' + process.env.NAME);
+console.log('Frontend plugin path: ' + process.env.PLUGIN_FULLPATH);
+
 module.exports = {
 
     // https://webpack.js.org/concepts/mode/
     mode: 'development',
 
+    plugins: [
+        new CopyPlugin({
+            patterns: [
+                // copy plugin frontend files frontend into a place where webpack can include them
+                // custom backend files can be accessed directly and do not need to be copied
+                {
+                    from: process.env.PLUGIN_FULLPATH + '/plugin/src/frontend/CustomStudentView.ts',
```

Comment:
Could we ensure that we have a volume mount in the Docker compose file to mount the Classy plugin into the Docker container so it is available to be copied? 

If so, would the PLUGIN_FULLPATH be a variable or a static path? Could we possibly read the static path to see if it contains a plugin? If so, then use it as a plug?